chapter-14/recipe-03/fortran-example/dashboard.cmake

```diff
@@ -1,32 +0,0 @@
-set(CTEST_PROJECT_NAME "example")
-set(CTEST_SITE "localhost")
-set(CTEST_BUILD_NAME "${CMAKE_SYSTEM_NAME}-${CMAKE_HOST_SYSTEM_PROCESSOR}")
-
-set(CTEST_SOURCE_DIRECTORY "${CTEST_SCRIPT_DIRECTORY}")
-set(CTEST_BINARY_DIRECTORY "${CTEST_SCRIPT_DIRECTORY}/build")
```

Comment:
The "build" fails because it appends a timestamp. I think we could remove the timestamp addition except when on "local".

Reply:
Ah wait, maybe not, I think I got confused.